### DISCUSSION TITLE
plane deployement not working because of plane.env

### DISCUSSION THREAD

**Discussion by YassineMajdoubDev** [makeplane/plane#4117] [General]
i created changed plane.env file and put my desired configuration and even when i don't change it and keep the default plane.env ,i still get the same issue :

WARN[0000] The "CORS_ALLOWED_ORIGINS" variable is not set. Defaulting to a blank string.

ERROR: The Compose file '/home/aftercode/plane-selfhost/plane-app/docker-compose.yaml' is invalid because:
services.proxy.ports contains an invalid type, it should be a number, or an object
services.migrator.restart contains an invalid type, it should be a string
Service 'web' is not running.

these are the logs of my containers all of them return the same issue 

![github](https://github.com/makeplane/plane/assets/163139536/8c588f38-ee77-4f8a-a73b-8461425dd9dc)

![image](https://github.com/makeplane/plane/assets/163139536/367adaf3-ed5f-45c6-af10-b1a5426b9c24)

can someone help me with this

**Comment by frenchcharly** [comment]
Encountered the same problem / message.

**Comment by iqbalpb** [comment]
Same Issue

**Comment by ramzsey** [comment]
Hello!
Is this still an issue which has not been fixed?
